keymap images

burkfers opened the pull request:
todo:
- [X] make drawing all boards in a pretty manner possible
- [x] Charybdises: Verify the layouts drawn are the stock `via` layouts
  - [x] Nano
  - [x] Mini
  - [x] Full size
- [x] Dactyls: Add the stock layouts in yaml
  - [x] Skeleyl
  - [x] TBK Mini
  - [x] Scylla
- [ ] ~~write a page that includes the images~~

Commit:
add keymap-drawer layouts and generated images for charybdis family

Commit:
add keymap-drawer instructions

Commit:
finish layouts for the Charybdises

Commit:
Add layouts for the dactyls

Commit:
Expand layout image instructions and add helper script

burkfers commented:
Companion PR for keymap-drawer support in flight: https://github.com/caksoylar/keymap-drawer/pull/76

Commit:
update cnano layout to fit bkb-master via keymap

Commit:
remove unneeded svg extra styles

Commit:
update cmini layout to fit bkb-master via keymap

Commit:
update charybdis layout to fit bkb-master via keymap

bstiq commented:
Once the keymap pictures are final, we should also:
- update the online docs to link to them (in the final steps)
- upload the keymap pictures to the product pages on the website (to be discussed internally)
- link to them on the main `readme.md` page of each repository

Commit:
update skeletyl layout to fit bkb-master default keymap

Commit:
update tbkmini layout to fit bkb-master default keymap

Commit:
update scylla layout to fit bkb-master default keymap

Commit:
spelling: expand wordlist

Unfortunately cspell does not care about code blocks in markdown.
Have to add some literals to the wordlist

Commit:
regenerate all keymap images

burkfers commented:
All done!

When reviewing, please pay special attention to the layer names of the dactyls. As all three only use numbered indices in the source and a single number as the layer name doesn't look very appealing in the image, I made up some names.

I excluded using the new images on any pages from this PR as the customizing section need other work as well, which would bloat the scope of this PR.

Commit:
correct charybdis highlighted keys and thumb transparent keys

Commit:
add script for generating single-layer keymap-drawer images

Commit:
improve layout rendering script

bstiq commented:
**Good to go on my side.** From my initial review of the images the layouts seem fine, if there's a few small mistakes they can be corrected later.

As discussed on Discord by @finrod09 :

> Leaving the images in the docs and linking there from outside makes the most sense to me, since the source to generate them lives there as well.

This also makes it simpler to update the images if/when needed (eg. the ongoing conversation about the placement of reset keycodes).

Now that those pictures are generated, we should include them somewhere in the docs. I opened a separate issue #17
